src/Sylius/Bundle/AttributeBundle/Twig/PercentageExtension.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AttributeBundle\Twig;
```

Comment:
We should put it in a UIBundle. It seems to be pretty useful for other bundles as well.